src/components/HeroSection.jsx

```diff
@@ -0,0 +1,129 @@
+import PropTypes from "prop-types";
+
+const HeroSection = ({
+  className,
+  backgroundSrc,
+  overlayPrimaryFillColor,
+  overlaySecondaryFillColor,
+  overlayTertiaryFillColor,
+  conferenceStatus,
+  taglineText,
+  sloganTextColor,
+  sloganText,
+  title,
+  missionText,
+  eventDetailsBgColor,
+  eventDetailsText,
+  verse,
+  verseReferenceColor,
+  verseReference,
+}) => {
+  return (
+    <div className={className}>
+      <div className="relative">
+        <div className="overflow-hidden absolute inset-0 -z-10 rounded-bl-[5rem] md:rounded-bl-[10rem] xl:rounded-bl-[15rem]">
+          <img
+            alt="Background image"
+            className="object-cover absolute opacity-75 size-full grayscale"
+            src={backgroundSrc}
+          />
+          <BaseOverlay
+            primaryFillColor={overlayPrimaryFillColor}
+            secondaryFillColor={overlaySecondaryFillColor}
+            tertiaryFillColor={overlayTertiaryFillColor}
+          />
+        </div>
+        <div className="flex flex-col gap-y-4 items-center px-4 pt-36 pb-10 text-center sm:pb-20 md:pt-44">
+          {conferenceStatus == "post-conference" && (
+            <p className="py-1 px-4 uppercase rounded-xl border-2 border-black w-fit text-h5">
+              Recap
+            </p>
+          )}
+          <p className="uppercase text-h5">{taglineText}</p>
+          <p className={`text-h5 ${sloganTextColor}`}>{sloganText}</p>
+          <h1 className="mx-10 uppercase text-h1">{title}</h1>
+          <p className="-mt-4 max-w-lg italic font-bold sm:-mt-6 sm:text-lg md:text-xl">
+            {missionText}
+          </p>
+          <p
+            className={`py-2 px-6 font-bold text-white rounded-xl sm:mt-6 w-fit text-p ${eventDetailsBgColor}`}
+          >
+            {eventDetailsText}
+          </p>
+        </div>
+      </div>
+      <p className="px-4 pt-10 mx-auto max-w-lg text-lg italic text-center">
+        {verse}
+      </p>
+      <p
+        className={`px-4 pb-10 text-lg italic font-bold text-center ${verseReferenceColor}`}
+      >
+        {verseReference}
+      </p>
+    </div>
+  );
+};
+
+HeroSection.propTypes = {
+  className: PropTypes.string,
+  backgroundSrc: PropTypes.string.isRequired,
+  overlayPrimaryFillColor: PropTypes.string.isRequired,
+  overlaySecondaryFillColor: PropTypes.string.isRequired,
+  overlayTertiaryFillColor: PropTypes.string.isRequired,
+  conferenceStatus: PropTypes.oneOf(["pre-conference", "post-conference"])
```

Comment:
nit: i think `pre` or `post` is enough as statuses

Reply:
i have it explicitly here because only `pre` and `post` can be ambiguous, it's not immediately clear what `pre` and `post` are referring to, could be `pre-planned`, etc.